jdaviz/configs/default/plugins/model_fitting/tests/test_blackbody.py

```diff
@@ -0,0 +1,173 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Tests for physical functions."""
```

Comment:
I am guessing you copied this from `astropy` somewhere. Maybe add a dev comment about that?

Also, in the future when astropy/astropy#12304 and astropy/astropy#12318 are resolved, what is going to happen to this code?

Reply:
Yes, this is copied from astropy PR 12304 (with slight modifications).  I think I had that documented in a commit message, but putting it in the code is a good idea - I've added a comment at the top of each file.

Once an eventual implementation for BlackBody with flux-units is finished in astropy (likely NOT 12304), we plan to replace the copy here with that implementation.

Reply:
Thanks! Yeah, if it is buried in commit message, one can only get to it via git log or blame, which is a bit tedious. 😅